legacy/charts/vela-core-legacy/crds/core.oam.dev_componentdefinitions.yaml

```diff
@@ -27,268 +27,532 @@ spec:
-                    type: string
-                  selector:
-                    additionalProperties:
-                      type: string
-                    description: Selector to select the child resources that the workload
-                      wants to expose to traits
-                    type: object
-                required:
-                - apiVersion
-                - kind
-                type: object
-              type: array
-            extension:
-              description: Extension is used for extension needs by OAM platform builders
-              type: object
-              
-            podSpecPath:
-              description: PodSpecPath indicates where/if this workload has K8s podSpec
-                field if one workload has podSpec, trait can do lot's of assumption
-                such as port, env, volume fields.
-              type: string
-            revisionLabel:
-              description: RevisionLabel indicates which label for underlying resources(e.g.
-                pods) of this workload can be used by trait to create resource selectors(e.g.
-                label selector for pods).
-              type: string
-            schematic:
-              description: Schematic defines the data format and template of the encapsulation
-                of the workload
-              properties:
-                cue:
-                  description: CUE defines the encapsulation in CUE format
+  version: v1alpha2
+  versions:
+  - name: v1alpha2
+    schema:
+      openAPIV3Schema:
+        description: ComponentDefinition is the Schema for the componentdefinitions
+          API
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: ComponentDefinitionSpec defines the desired state of ComponentDefinition
+            properties:
+              childResourceKinds:
+                description: ChildResourceKinds are the list of GVK of the child resources
+                  this workload generates
+                items:
+                  description: A ChildResourceKind defines a child Kubernetes resource
+                    kind with a selector
```

Comment:
why the crd has so much difference in this PR?

Reply:
fix it